common/src/main/java/org/jboss/hal/testsuite/util/ElytronIntegrationOperations.java

```diff
@@ -0,0 +1,262 @@
+package org.jboss.hal.testsuite.util;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.Property;
+import org.jboss.hal.testsuite.creaper.ResourceVerifier;
+import org.jboss.hal.testsuite.dmr.ModelNodeGenerator;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.jboss.hal.testsuite.fragment.formeditor.Editor;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+import java.io.IOException;
+
+/**
+ * Class which serves as an abstraction over modifying and verifying credential-reference attributes throughout HAL
+ */
+public class ElytronIntegrationOperations {
```

Comment:
I would suggest to change this class name since we usually use `Operations` suffix for classes related only to model operations. What about `ElytronIntegrationChecker` or do you have some other idea?